frontend/app/src/services/rotkehlchen-api.ts

```diff
@@ -103,11 +104,17 @@ export class RotkehlchenApi {
    * @return The validity of the status code
    */
   private static modifyWithExternalService(status: number): boolean {
-    return status === 200 || status === 400 || status === 409 || status == 502;
+    return (
+      status === 200 ||
+      status === 400 ||
+      status === 409 ||
+      status === 500 ||
+      status === 502
+    );
   }
 
   private static fetchWithExternalService(status: number): boolean {
-    return status === 200 || status === 409 || status == 502;
+    return status === 200 || status === 409 || status === 500 || status === 502;
```

Comment:
why are we addint 500 to the valid statuses?

Reply:
Just following what the API docs indicate as potential status. Should 500 not be in there? If it's an actual potential response from the API it's just as valid as 502, no?
![image](https://user-images.githubusercontent.com/27592/84110263-d1957900-aa24-11ea-895c-0c4ff479d078.png)

Reply:
We don't need to add 500 there. The others still return a `{ result: .., message: '' }` son that we might need to parse that this why they are added there. For 500 it doesn't matter.